This has been submited to the TNP community as bug 3685:
https://bugs.ntp.org/show_bug.cgi?id=3685 
This patch may be removed when upgrading to a version of NTP with this bug
already resolved.

--- ntpd/ntpd.c
+++ ntpd/ntpd.c
@@ -916,7 +916,7 @@ ntpdmain(
 	msyslog(LOG_NOTICE, "----------------------------------------------------");
 	msyslog(LOG_NOTICE, "ntp-4 is maintained by Network Time Foundation,");
 	msyslog(LOG_NOTICE, "Inc. (NTF), a non-profit 501(c)(3) public-benefit");
-	msyslog(LOG_NOTICE, "corporation.  Support and training for ntp-4 are");
+	msyslog(LOG_NOTICE, "corporation.  Information and training for ntp-4 are");
 	msyslog(LOG_NOTICE, "available at https://www.nwtime.org/support");
 	msyslog(LOG_NOTICE, "----------------------------------------------------");
 
